### PR TITLE
updating some type hints and also updating a grammatic error for tile…

### PR DESCRIPTION
**Pull request by jtsodapop**
…d imagery

Note: everything in the Data folder has been checked over. orm I am unsure if these need updating, so leaving behind. Will be looking to look through the Schema folder next to complete this

### CODE REVIEW

File: labelbox/data/serialization/labelbox_v1/objects.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class LBV1ObjectBase(LBV1Feature):
     classifications: List[Union[LBV1Text, LBV1Radio, LBV1Dropdown,
                                 LBV1Checklist]] = []
 
-    def dict(self, *args, **kwargs):
+    def dict(self, *args, **kwargs) -> Dict[str, str]:
```
Comment (msokoloff1) [inline labelbox/data/serialization/labelbox_v1/objects.py:26]: this is `Dict[str, Any]` since all of these objects will be nested json objects.

**Comment by msokoloff1** [comment]
I know that there aren't any circuluar imports here but it still feels dangerous. Do you mind merging your PR to reduce the imports into this PR before we merge this?

**Comment by jtsodapop** [comment]
> I know that there aren't any circuluar imports here but it still feels dangerous. Do you mind merging your PR to reduce the imports into this PR before we merge this?

Yes I can do that

**Comment by jtsodapop** [comment]
@msokoloff1  Looks like the data row creation issue is persisting so I can't confirm the checks on github or locally, but updates have been made !

**Review by msokoloff1 — APPROVED** [review]

**Pull request merged**